### PR TITLE
add possibility for predefined api key

### PR DESCRIPTION
**Pull request by mvdbeek**
This should eliminate the necessity to have another script in https://github.com/galaxyproject/ansible-galaxy-extras/blob/master/templates/create_galaxy_user.py.j2 that does almost the same thing.
I reduced the minimum password length to 5, since @bgruening and me are using admin as our preset password.

### CODE REVIEW

**Comment by mvdbeek** [comment]
Ping @afgane @bgruening, can you guys have a look?

File: defaults/main.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ galaxy_tools_ignore_errors: true
 # A system path from where this role will be run
 galaxy_tools_base_dir: /tmp
 
+# Set this to yes and change galaxy_tools_api_key to a string to create a predefined api key for the bootstrap user.
+galaxy_tools_admin_user_preset_api_key: no
```
Comment (bgruening) [inline defaults/main.yml:23]: Can we stick to True/False?

File: files/manage_bootstrap_user.py [hunk 2]
```diff
@@ -196,12 +203,12 @@ def get_bootstrap_app(ini_file):
     return app
 
 
-def create_bootstrap_user(ini_file, username, user_email, password):
+def create_bootstrap_user(ini_file, username, user_email, password, preset_api_key):
```
Comment (bgruening) [inline files/manage_bootstrap_user.py]: This should default to False I think.

File: defaults/main.yml (hunk 1 above)
Comment (mvdbeek) [inline defaults/main.yml:23]: Either way is fine with me, but @afgane prefers yes/no (and that's consistent within this file).

**Comment by bgruening** [comment]
:+1: from my side.

**Comment by mvdbeek** [comment]
I'm gonna go ahead and merge this, if there are any problems when you guys are updating the role let me know!

**Pull request merged**